src/gt4py/next/program_processors/runners/gtfn_cpu.py

```diff
@@ -20,9 +20,9 @@
 from gt4py.eve.utils import content_hash
```

Comment:
let's rename the module to `gtfn`

Reply:
I wanted to, but `next.program_processors.runners.gtfn` clashes with `next.program_processors.codegens.gtfn`, so we'll have to find a different name.